output/influxdb/util.go

```diff
@@ -18,12 +20,21 @@ func MakeClient(conf Config) (client.Client, error) {
 	if conf.Addr.String == "" {
 		conf.Addr = null.StringFrom("http://localhost:8086")
 	}
+	var ProxyURL *url.URL
+	if conf.Proxy.Valid && conf.Proxy.String != "" {
+		parsedProxyURL, err := url.Parse(conf.Proxy.String)
+		if err != nil {
+			return nil, err
+		}
+		ProxyURL = parsedProxyURL
+	}
 	return client.NewHTTPClient(client.HTTPConfig{
 		Addr:               conf.Addr.String,
 		Username:           conf.Username.String,
 		Password:           conf.Password.String,
 		UserAgent:          "k6",
 		InsecureSkipVerify: conf.Insecure.Bool,
+		Proxy:              http.ProxyURL(ProxyURL),
```

Comment:
This is seems strange -  I don't know if `http.ProxyURL` works with an empty URL or if the client handles it - but it seems strange ot set it if it is *not* valid.

I would prefer that the `client.HTTPConfig` is made and tehn the proxyURL is set only if ti is configured and then use that config at the end again to make the new client.

Reply:
I tested this code with K6_INFLUXDB_PROXY set and unset, worked both ways without problems.

Reply:
I am going to check that parsing

Reply:
@mstoykov saparatetd client.HTTPConfig

Reply:
I am not sure how to parse the Proxy field since it goes to the transport, not to the URL.

Reply:
I share the same idea as @mstoykov, I would prefer not to rely on InfluxDB-client internal logic. I prefer if we stay conservative here. In this way, we make clear to the next contributor what the logic is.